content/en/account_management/teams.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Teams
+kind: documentation
+---
+
+## Overview
+Datadog Teams allow groups of users to organize their team assets within Datadog and automatically filter their Datadog-wide experience to prioritize these assets.
+
+Use Teams to link resources such as dashboards, services, monitors, and incidents to a group of users. You can also add team-specific links to Slack channels, Jira boards, and Github repositories.
+
+Team membership is flexible. Users can join teams or be added by an administrator. Users can belong to multiple teams. 
+
+## Setup
+
+### Navigation
+
+To find the [Team directory page][1]: 
```

Comment:
just spent way too long thinking about whether "team directory page" should be capitalized. i am feeling that it should be a lowercase "team" but this is a soft recommendation, it's up to you